poolapi/apijson.go

```diff
@@ -13,31 +13,32 @@ type Response struct {
 // TODO: make JSON tags lower-case and add "_" between words
 
 type PurchaseInfo struct {
-	PoolAddress   string `json:"PoolAddress"`
-	PoolFees      string `json:"PoolFees"`
-	Script        string `json:"Script"`
-	TicketAddress string `json:"TicketAddress"`
+	PoolAddress   string  `json:"PoolAddress"`
+	PoolFees      float64 `json:"PoolFees"`
```

Comment:
Actually I found my code to do the conversion client side.  If you think it makes sense to switch then please do so (and on the pool list at https://decred.org/api/?c=gsd) otherwise i'll continue using my converter.